JenkinsService/src/main/java/com/revature/JenkinsServlet.java

```diff
@@ -86,10 +100,12 @@ void makeJob() throws IOException {
         try {
             exitCode = process.waitFor();
         } catch (InterruptedException e) {
+            log.error("Exception occurred within makeJob method.  " + e.getMessage());
             throw new IOException(e.getMessage());
         }
 
         if (exitCode != 0) {
+            log.error("makeJob method failing due to cURL failure, exitCode=0");
```

Comment:
The exitCode is _not_ 0, which is why it's failing

Reply:
corrected